lighthouse.go

```diff
@@ -711,15 +711,22 @@ func (lh *LightHouse) unlockedShouldAddV6(vpnAddr netip.Addr, to *V6AddrPort) bo
 }
 
 func (lh *LightHouse) IsLighthouseAddr(vpnAddr netip.Addr) bool {
-	_, ok := lh.GetLighthouses()[vpnAddr]
-	return ok
+	l := lh.GetLighthouses()
+	for i := range l {
```

Comment:
Might be ever so slightly slower but `slices.Contains` is there.

Reply:
it's meaningfully slower, at least in my testing :sob: